java/src/main/native/src/AggregationJni.cpp

```diff
@@ -81,17 +81,19 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_Aggregation_createNoParamAgg(JNIEnv
       case 17: // ROW_NUMBER
         ret = cudf::make_row_number_aggregation();
         break;
-      // case 18: COLLECT_LIST
-      // case 19: COLLECT_SET
-      // case 20: MERGE_LISTS
-      case 20:
+      // case 18: RANK
+      // case 19: DENSE_RANK
+      // case 20: COLLECT_LIST
+      // case 21: COLLECT_SET
+      // case 22: MERGE_LISTS
+      case 22:
         ret = cudf::make_merge_lists_aggregation();
         break;
-      // case 21: MERGE_SETS
-      // case 22: LEAD
-      // case 23: LAG
-      // case 24: PTX
-      // case 25: CUDA
+      // case 23: MERGE_SETS
+      // case 24: LEAD
```

Comment:
Lines 223 and 226 of this file need to also be updated because you changed what id LEAD and LAG use.